gantry/commands/run.py

```diff
@@ -473,11 +473,24 @@ def run(
         cl_objects = beaker.cluster.list()
         final_clusters = []
         for pat in cluster:
-            matching_clusters = [cl.full_name for cl in cl_objects if fnmatch(cl.full_name, pat)]
+            matching_clusters = []
+            for cl in cl_objects:
+                if fnmatch(cl.full_name, pat):
+                    matching_clusters.append(cl.full_name)
+                elif cl.aliases:
+                    org_name = cl.full_name.split("/")[0]
+                    for alias in cl.aliases:
+                        if "/" not in alias:
```

Comment:
Aliases won't have slashes, so this check isn't needed

Reply:
[4de3222](https://github.com/allenai/beaker-gantry/pull/153/commits/4de322219687d317f322a66860ef2c8de82108ab)